pkg/cmd/operator/cmd.go

```diff
@@ -0,0 +1,18 @@
+package operator
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/cluster-csi-snapshot-controller-operator/pkg/operator"
+	"github.com/openshift/cluster-csi-snapshot-controller-operator/pkg/version"
+	"github.com/openshift/library-go/pkg/controller/controllercmd"
+)
+
+const componentName = "csi-snapshot-controller-operator"
+
+func NewOperator() *cobra.Command {
+	cmd := controllercmd.NewControllerCommandConfig(componentName, version.Get(), operator.RunOperator).NewCommand()
+	cmd.Use = "operator"
+	cmd.Short = "Start the CSI Snapshot Controller Operator"
+	return cmd
```

Comment:
Why does this operator have two cmds?

Reply:
I've removed the content under `pkg/cmd/`, keeping the main function under `cmd/`.